goon/code/datums/browserOutput.dm

```diff
@@ -98,12 +98,20 @@ For the main html chat area
 			data = analyzeClientData(arglist(params))
 
 		if("encoding")
+			var/encoding = href_list["encoding"]
 			var/static/regex/RE = regex("windows-(874|125\[0-8])")
-			if (RE.Find(href_list["encoding"]))
-				world.log << "ENCODING RECEIVED: [href_list["encoding"]]"
+			if (RE.Find(encoding))
 				owner.encoding = RE.group[1]
+
+			else if (encoding == "gb2312")
```

Comment:
maybe add a comment or something so it isn't an arcane thing people have to google every time they want to know what encoding it is